dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/transform/DotTransformerBuilder.java

```diff
@@ -125,6 +125,17 @@ public DotTransformerBuilder keyValueToMapTransformer(){
         return this;
     }
 
+    /**
+     * Url Content Map to Map Transformer
+     * @return
+     */
+    public DotTransformerBuilder urlContentMapTransformer(){
```

Comment:
I not understand what it is this method doing, I think it need a better doc

Reply:
I am just following the same convention that is already done by @fabrizzio-dotCMS